### PR TITLE
Add label fields

### PR DESCRIPTION
**Pull request by kamaleshnneerasa**
## Description
Please include a summary of the change, motivation and context.

### Testing
Please describe the tests that you ran to verify your changes. Please summarize what did you test and what needs to be tested e.g. deployed and tested helm chart locally. 

### Checklist:
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Any dependent changes have been merged and published in downstream modules

### Documentation
Make sure that you have documented corresponding changes in this repository or [hypertrace docs repo](https://github.com/hypertrace/hypertrace-docs-website) if required.

### CODE REVIEW

File: labels-config-service-api/src/main/proto/org/hypertrace/label/config/service/v1/label.proto [hunk 1]
```diff
@@ -7,8 +7,17 @@ option java_multiple_files = true;
 message Label {
   string id = 1;
   string key = 2;
+  optional string created_by = 3;
```
[the remark this answers is not in the document]
Comment (kamaleshnneerasa) [reply labels-config-service-api/src/main/proto/org/hypertrace/label/config/service/v1/label.proto]: done.

File: labels-config-service-impl/src/main/java/org/hypertrace/label/config/service/LabelsConfigServiceImpl.java [hunk 2]
```diff
@@ -172,28 +176,28 @@ public void getLabels(
   public void updateLabel(
       UpdateLabelRequest request, StreamObserver<UpdateLabelResponse> responseObserver) {
     try {
-      Label updatedLabelInReq = request.getLabel();
+      LabelData newLabelData = request.getData();
```
[the remark this answers is not in the document]
Comment (kamaleshnneerasa) [reply labels-config-service-impl/src/main/java/org/hypertrace/label/config/service/LabelsConfigServiceImpl.java]: Done.

**Review by aaron-steinfeld — COMMENTED** [review]
lgtm; let's hold on merge until corresponding work is done in GQL as this is a breaking change and we want to minimize the window of inconsistency.

**Comment by aaron-steinfeld** [comment]
Let's fix snyk then get this merged.

**Review by aaron-steinfeld — APPROVED** [review]

**Pull request merged**